test/util/test_code_extraction_util.py

```
@@ -61,7 +61,7 @@ def test_extract_specs_empty_specs() -> None:
         "postconditions": []
     }
     """
-    assert code_extraction_util.parse_specs(text) == None, (
+    assert code_extraction_util.parse_clauses_for_spec(text) == None, (
```

Ruff (0.14.14)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)